test/src/Test/Main.purs

```diff
@@ -133,14 +136,24 @@ main = void $ runAff throwException (const (pure unit)) $ jumpOutOnError do
     run isRight $ map show <$> QF.getMetastore
 
     log "\nReadQuery:"
-    run isRight $ QF.readQuery Json.Readable testDbAnyDir "SELECT sha as obj FROM `/slamengine_commits.json`" (SM.fromFoldable [Tuple "foo" "bar"]) (Just { offset: 0, limit: 1 })
-    run isRight $ QF.readQuery Json.Precise testDbAnyDir "SELECT sha as obj FROM `/slamengine_commits.json`" (SM.fromFoldable [Tuple "foo" "bar"]) (Just { offset: 0, limit: 1 })
+    run isRight $ QF.readQuery Json.Readable testDbAnyDir
+      (unsafePartial $ fromParsed "ReadQueryA" $ Sql.parseQuery "SELECT sha as obj FROM `/slamengine_commits.json`")
+      (SM.fromFoldable [Tuple "foo" "bar"])
+      (Just { offset: 0, limit: 1 })
+    run isRight $ QF.readQuery Json.Precise testDbAnyDir
+      (unsafePartial $ fromParsed "ReadQueryB" $ Sql.parseQuery "SELECT sha as obj FROM `/slamengine_commits.json`")
```

Comment:
I'd just make an `unsafeQuery` for these instead that's `unsafePartial $ fromRight $ parseQuery` - it's not that hard to figure out where these things fail when they do, and would avoid a bunch of repetition.